src/Regridder.jl

```diff
@@ -507,6 +507,9 @@ function update_surface_fractions!(cs::CoupledSimulation)
     cs.surface_fractions.ice .= max.(min.(ice_d, FT(1) .- land_s), FT(0))
     cs.surface_fractions.ocean .= max.(FT(1) .- (cs.surface_fractions.ice .+ land_s), FT(0))
 
+    comms_ctx = axes(land_s).grid.topology.context
```

Comment:
```suggestion
    comms_ctx = ClimaComms.context(land_s)
```
I think we have a helper for this, so that you don't need to use internals.